single-line-diagram/single-line-diagram-core/src/test/java/com/powsybl/sld/iidm/TestCase11SubstationGraph.java

```diff
@@ -138,11 +139,9 @@ void testHorizontalDefaultStyle() {
         // (with horizontal substation layout)
         SubstationGraph substationGraph = graphBuilder.buildSubstationGraph(substation.getId());
 
-        assertTrue(compareMetadata(substationGraph, "/substDiag_metadata.json",
-                new HorizontalSubstationLayoutFactory(),
-                new PositionVoltageLevelLayoutFactory(),
-                new DefaultDiagramLabelProvider(network, componentLibrary, layoutParameters),
-                new BasicStyleProvider()));
+        DefaultSVGWriter defaultSVGWriter = new DefaultSVGWriter(componentLibrary, layoutParameters, svgParameters);
```

Comment:
This line of code is added everywhere. Wasn't it possible to create it in  compareMetadata for instance to keep code simple?

Reply:
I removed those lines. The `DefaultSvgWriter` is now created in a draw function, in the `SingleLineDiagram` class.